### PR TITLE
TECH-1254 debug strict mode

### PR DESCRIPTION
**Pull request by ericbroder**

### CODE REVIEW

**Comment by ericbroder** [comment]
testing on QA

File: scripts/install_packages.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-# set -eo pipefail
-# IFS=$'\n\t'
+set -eo pipefail
+IFS=$'\n\t'
 
 echo "Start installing packages"
 cd /home/ubuntu/ourrevolution
 source /home/ubuntu/.virtualenvs/ourrevolution/bin/virtualenvwrapper.sh
-workon ourrevolution
+workon ourrevolution || true
```
Comment (cjmabry) [inline scripts/install_packages.sh]: Why do we need `|| true` here? If environment isn't available, isn't something wrong?

Comment (ericbroder) [reply scripts/install_packages.sh]: https://github.com/Our-Revolution/site/pull/275/files

Comment (ericbroder) [reply scripts/install_packages.sh]: https://github.com/Our-Revolution/site/pull/275

**Pull request merged**